helm/athena/templates/_helpers.tpl

```diff
@@ -1,3 +1,29 @@
+{{- define "athena.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "athena.name" -}}
+athena
+{{- end -}}
+
+{{- define "athena.labels.selector" -}}
+app: {{ include "athena.name" . }}
+app.kubernetes.io/name: {{ include "athena.name" . }}
+app.kubernetes.io/component: {{ include "athena.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name | quote }}
+{{- end -}}
+
+{{- define "athena.labels.common" -}}
```

Comment:
There were no labels added to the helm chart other than `App: athena` so I defined a couple here.